app/src/main/java/com/pvh98/calculator/CalculatorFrg.kt

```diff
@@ -0,0 +1,217 @@
+package com.pvh98.calculator
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import kotlinx.android.synthetic.main.act_main.*
+import kotlinx.android.synthetic.main.frg_calculator.*
+
+class CalculatorFrg : Fragment(), View.OnClickListener {
+    var onOpeDecimal = false
+    var mValueOne = 0.0f
```

Comment:
```suggestion
    var firstValue = 0.0f
```
Tên biến trong Kotlin không cần phải thêm tiền tố. Em có thể nghiên cứu thêm ở đây:
https://kotlinlang.org/docs/reference/coding-conventions.html